humanapi/connect-client
# COR-4230 & COR-4261: Ensure focus & meaningful title to iframe for WCAG 2.1

## What did you implement
- add `title` attribute to `iframe`
- focus to iframe after launching with `focus()` method 
- ~~`aria-hidden` to `app` and `footer` (if any) to hide from accessibility API when Connect is launched~~ moved logic out of library
- bump to 2.2.1-beta.4

## How can we verify it
- integrate Connect following this guide [https://reference.humanapi.co/docs/web-guide](https://reference.humanapi.co/docs/web-guide)
-- Include [https://cdn.humanapi.co/humanapi-connect-client@2.2.1-beta.4.js](https://cdn.humanapi.co/humanapi-connect-client@2.2.1-beta.4.js) in your code OR `npm i humanapi-connect-client@2.2.1-beta.4` in your project
- after launching Connect, focus should be on `iframe`
-- If using Google Chrome, you can verify using the `Create Live Expression` (the eyeball icon) in DevTools and adding `document.activeElement` in the console
-- pressing tab will go to the next item in the iframe
